modules/social_features/social_comment/social_comment.module

```diff
@@ -360,3 +360,24 @@ function social_comment_field_info_alter(&$info) {
     $info['comment']['list_class'] = '\Drupal\social_comment\SocialCommentFieldItemList';
   }
 }
+
+/**
+ * Implements hook_preprocess_pager().
+ */
+function social_comment_preprocess_pager(&$variables) {
+  $items = &$variables['items'];
+  $pattern = '/%2C.$/';
+
+  // Change href for all pager items except "pages"
+  // (E.G. first, previous, next, last, etc.).
+  foreach ($items ?? [] as $key => $item) {
```

Comment:
Need to add check if $items are not empty because sometimes it throws errors.

Reply:
In this case, items can not be empty, because they exist or not, so it's can be null or array.

Reply:
Fixed.